src/helpers/GaugeVoterSnapshotter.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.29;
+
+import { DaoAuthorizableUpgradeable } from "@aragon/commons/permission/auth/DaoAuthorizableUpgradeable.sol";
+import { IDAO } from "@aragon/commons/dao/IDAO.sol";
+import { IGaugeVoterSnapshotter } from "../interfaces/helpers/IGaugeVoterSnapshotter.sol";
+import { IAddressGaugeVoter } from "../interfaces/helpers/IAddressGaugeVoter.sol";
+
+/// @title GaugeVoterSnapshotter
+/// @notice Contract that captures and stores historical gauge voting data
+/// @dev Stores snapshots of gauge votes and total voting power for each epoch
+contract GaugeVoterSnapshotter is IGaugeVoterSnapshotter, DaoAuthorizableUpgradeable {
```

Comment:
Why using `DaoAuthorizableUpgradeable` if there is no `auth()` guard or `dao()` getter?